Samples/3.3/Streaming/CustomDataAdapter/Grains/ConsumerGrain.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Threading.Tasks;
+using Common;
+using GrainInterfaces;
+using Microsoft.Extensions.Logging;
+using Orleans;
+using Orleans.Streams;
+using Orleans.Streams.Core;
+
+namespace Grains
+{
+    [ImplicitStreamSubscription(Constants.StreamNamespace)]
```

Comment:
Maybe ad a comment to highlight the use of implicit subscription and explain what that means?